src/core/server/ui_settings/utils.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { UiSettingScope } from './types';
+
+export const CURRENT_USER_PLACEHOLDER = '<current_user>';
```

Comment:
I have an impression that the current `request` can injected into uiSettingClient when I was doing the investigation,  I will take another look, but I think that won't block this PR.